### PR TITLE
Created a new page to articulate PCI shared responsibilities

### PR DESCRIPTION
**Pull request by rinchen**
Created a new page to articulate PCI shared responsibilities for compliance with PCI section 12

### CODE REVIEW

File: src/security/pci-guidance.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# PCI DSS Responsibility Guidance
+
+Platform.sh has many PCI certified customers using our services. Below we will attempt to clarify the PCI compliance responsibilities for Platform.sh and our customers.
```
Comment (Crell) [inline src/security/pci-guidance.md]: I heartily dislike "below we will" type language.  Recommended alternative:

"Some aspects of PCI compliance are the responsibility of the host, described below.  Others are the responsibility of the application developer."

Comment (Crell) [reply src/security/pci-guidance.md]: (That's just for the second sentence.  The first sentence is fine, but frankly extraneous.)

File: src/security/pci-guidance.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+Simple audit questions can be handled via support tickets however in-depth support for your audit may required additional arrangements. Please contact your Platform.sh account manager to discuss your needs.
```
Comment (Crell) [inline src/security/pci-guidance.md]: Suggested verbage: "Basic PCI questions can be handled by our support team via a ticket.  For more advanced questions or walkthrough of a full audit please contact your Platform.sh Account Manager.

File: src/security/pci-guidance.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+
+## Overview
+
+Platform.sh provides a Platform as a Service (PaaS) solution which our customers may use to store, process, or transmit cardholder data.
+
+
+## Security & Compensating Controls
+
+ * For those customers needing to meet SAQ C or SAQ D, a Platform.sh Enterprise Plan may be needed in order to fully meet your needs. Please [contact us](https://platform.sh/contact) for more information.
```
Comment (Crell) [inline src/security/pci-guidance.md]: No leading space on the bullets.  I... don't now if you want them to be separate UL blocks or not.

File: src/security/pci-guidance.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+ * Please take note that customer environments are deployed in a read-only instance, segregated with GRE and IPSEC tunnels, which often permits compensating controls to be claimed for several PCI requirements.
+
+ * Because customers can use our PaaS in a variety of ways, the best approach with auditors is to focus is on “What do I, the customer, control/configure and how is it managed in a compliant manner?”
+
+
+## Responsibility
+
+Platform.sh customers are ultimately responsible for their own PCI DSS compliance.
+
+The following sections provide guidance on shared responsibilities to achieve PCI DSS compliance. We have chosen to list all the sections found in a typical SAQ D although you may not need this level of detail for your efforts. This section is currently using PCI DSS 3.2 as a reference.
```
Comment (Crell) [inline src/security/pci-guidance.md]: Please reword this to avoid the need for the royal we if you can.  Like, "This list below covers all sections found..."

File: src/security/pci-guidance.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+### Requirement 1
+
+|Requirement|Responsibility|Comments|
+|---|---|---|
+|1.1 to 1.3.3| Platform|Infrastructure cannot be modified by the customer|
+|1.3.4|Shared|Platform is responsible for the infrastructure. Customer is responsible for correct routes.yaml configuration|
+|1.3.5 to 1.3.6|Platform|Infrastructure cannot be modified by the customer|
+|1.3.7|Shared|Customers are responsible for ensuring their application does not leak private IP addresses|
+|1.4 to 1.5|Shared|Customers are responsible for their devices accessing the environment|
```
Comment (Crell) [inline src/security/pci-guidance.md]: I know it's not necessary for markdown proper, but please format the table in the source so it's readable.

**Comment by rinchen** [comment]
@Crell I reformatted the tables as requested. Good call as I found two errors which the MD engine I'm using to test ignored. The MD documentation for table headers (i.e. "|---|" that I read indicates it has to be like this and not expanded. I'm unsure if gitbook will render this correctly if I space it out.

**Comment by Crell** [comment]
It will definitely render nicely if you have the header line full-width.  All of the other tables in the docs do that.  (Of which we have only a few, granted.)
